packages/create-turbo/src/utils/isFolderEmpty.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs-extra";
+
+const VALID_FILES = [
+  ".DS_Store",
+  ".git",
+  ".gitattributes",
+  ".gitignore",
+  ".gitlab-ci.yml",
+  ".hg",
+  ".hgcheck",
+  ".hgignore",
+  ".idea",
+  ".npmignore",
+  ".travis.yml",
+  "LICENSE",
+  "Thumbs.db",
+  "docs",
+  "mkdocs.yml",
+  "npm-debug.log",
+  "yarn-debug.log",
+  "yarn-error.log",
+  "yarnrc.yml",
+  ".yarn",
+];
+
+export function isFolderEmpty(root: string): {
```

Comment:
I missed where this is used, but the `VALID_FILES` seems like a heuristic that's just waiting to break

Reply:
This is also from create-next-app